ansible/deploy.yml

```diff
@@ -128,6 +128,12 @@
         notify: "restart aeternity daemon"
         tags: [package]
 
+      - name: Fail on missing 'node_config.chain.db_path' variable
+        fail:
+          msg: 'Chain persistence is enabled, but "node_config.chain.db_path" is not defined'
+        when: node_config.chain.persist is defined and node_config.chain.persist and node_config.chain.db_path is not defined
```

Comment:
could you please split this in multiple line for readability ? (when takes a list)

Reply:
https://github.com/aeternity/infrastructure/pull/401/commits/eee3fdbf274424c0a165828060f3b4a1c8be2bd3